assertpy/assertpy.py

```diff
@@ -44,7 +45,7 @@ def contents_of(f):
     if type(f) is str:
         with open(f, 'r') as fp:
             contents = fp.read()
-    elif type(f) is file:
+    elif hasattr(f, 'read'):
```

Comment:
the duck type approach is usually taken in python. also file type doesn't exist in py3